pydoctor/epydoc2stan.py

```diff
@@ -74,11 +77,23 @@ def taglink(o: model.Documentable, page_url: str, label: Optional["Flattenable"]
         ret(title=o.fullName())
     return ret
 
-
+# TODO: Move this code to it's own module to avoid cyclic imports.
```

Comment:
Is this TODO for this PR, or is there a follow-up issue created to track its progress?

Reply:
Another PR, It's pretty simple change.

Reply:
I'll create an issue.